From 6feea32bcadcd78393bdb092af9c935e836eca7c Mon Sep 17 00:00:00 2001
From: Peng Fan <peng.fan@nxp.com>
Date: Fri, 14 Sep 2018 21:28:54 +0800
Subject: [PATCH 4716/5242] MLK-19664-1 ARM64: dts: domu: enable multi-vdev on
 one rpmsg channel

commit  d3617b37fcb456b3da799e7cbbb1c8e8d0f33d2c from
https://source.codeaurora.org/external/imx/linux-imx.git

Support multi-vdev on one rpmsg channel on 8QM DomU.
sync with commit f7f11791135f3f5("MLK-19558-2 ARM64: dts: enable multi-vdev on one rpmsg channel")

Signed-off-by: Peng Fan <peng.fan@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 .../boot/dts/freescale/fsl-imx8qm-mek-domu.dts     |    6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8qm-mek-domu.dts b/arch/arm64/boot/dts/freescale/fsl-imx8qm-mek-domu.dts
index ee91108..66cd7f5 100644
--- a/arch/arm64/boot/dts/freescale/fsl-imx8qm-mek-domu.dts
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8qm-mek-domu.dts
@@ -90,7 +90,7 @@
 
 		rpmsg_reserved: rpmsg@0x90100000 {
 			no-map;
-			reg = <0 0x90100000 0 0x10000>;
+			reg = <0 0x90100000 0 0x20000>;
 		};
 
 		/* global autoconfigured region for contiguous allocations */
@@ -542,8 +542,8 @@
 	/*
 	 * 64K for one rpmsg instance:
 	 */
-	vdev-nums = <1>;
-	reg = <0x0 0x90100000 0x0 0x10000>;
+	vdev-nums = <2>;
+	reg = <0x0 0x90100000 0x0 0x20000>;
 	status = "okay";
 };
 
-- 
1.7.9.5

